db/migrate/20161124041953_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :email
+      t.string :password_digest
```

Comment:
this is the only instance of password_digest in this code and I've never seen the term before. Can you explain it?